test/functional/tests/example/example_test.py

```diff
@@ -147,3 +149,62 @@ def test_example_multidut():
                 TestRun.LOGGER.info(dut1_ex.run_expect_success("which casctl").stdout)
                 for name, disk in TestRun.disks.items():
                     TestRun.LOGGER.info(f"{name}: {disk.path}")
+
+
+@pytest.mark.multidut(2)
+def test_drbd_example():
+    """
+        title: Example test using DRBD API.
+        description: Create primary and secondary resources on two DUTs using drbd.
+        pass_criteria:
+          - primary drbd resource created.
+          - secondary drbd resource created.
+    """
+    with TestRun.step("Prepare"):
+        dut1, dut2 = TestRun.duts
+        install_drbd()
+        nodes = []
+        for dut in TestRun.duts:
+            with TestRun.use_dut(dut):
+                TestRun.dut.hostname = TestRun.executor.run_expect_success('uname -n').stdout
+                TestRun.LOGGER.info(TestRun.dut.hostname)
+                for name, disk in TestRun.disks.items():
+                    TestRun.LOGGER.info(f"{name}: {disk.path}")
+                # TODO: retrieve disk path (/dev/nvme2n1) from config file?
+                nodes.append(Node(TestRun.dut.hostname, "/dev/nvme2n1", "/dev/sdb", dut.ip, "7790"))
```

Comment:
You have to use `disk.path` instead of string literals

Reply:
I suggest using ram disks for drbd metadata. I believe we have ram disk api integrated in test-framework already.